contents/blog/posthog-cloud-eu.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2022-10-10
+title: Introducing PostHog Cloud EU
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Product updates”]
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/posthog-eu-blog.png
+featuredImageType: full
+--
+
+It's no secret that some popular analytics tools, such as Google Analytics, have fallen foul of GDPR regulations in the EU. That's one reason why we've enabled users to self-host PostHog, or to anonymize user data in a way which complies with GDPR. 
```

Comment:
@joethreepwood I merged but I removed the link to gaisillegal here. I'd rather not send people off somewhere else if we can help it.